src/Main.hs

```diff
@@ -0,0 +1,27 @@
+import           Options.Applicative             (Parser, ParserInfo,
+                                                  execParser, fullDesc, header,
+                                                  help, helper, info, long,
+                                                  metavar, progDesc, short,
+                                                  strOption, value, (<>))
+import           Text.Pandoc.Diagrams
+import           Text.Pandoc.JSON
+
+main :: IO ()
+main = do
+   --opts <- execParser withHelp
```

Comment:
Why remove the option parsing?  If so, why keep the `optparse-applicative` dependency at all?